core/src/avm1/globals/style_sheet.rs

```diff
@@ -0,0 +1,207 @@
+use crate::avm1::object::{Object, TObject};
+use crate::avm1::property_decl::define_properties_on;
+use crate::avm1::{
+    property_decl::Declaration, ArrayObject, ExecutionReason, NativeObject, ScriptObject,
+};
+use crate::avm1::{Activation, Error, Value};
+use crate::avm1_stub;
+use crate::context::GcContext;
+use crate::html::{transform_dashes_to_camel_case, CssStream, TextFormat};
+use crate::string::AvmString;
+use gc_arena::Gc;
+
+const PROTO_DECLS: &[Declaration] = declare_properties! {
```

Comment:
Flash docs say that StyleSheet is version-gated to FP7, should these properties be marked as version-gated?

Reply:
Done!